database/src/migrations/20230719000000_update_funding_sources.ts

```diff
@@ -0,0 +1,625 @@
+  INSERT INTO INVESTMENT_ACTION_CATEGORY (agency_id, record_effective_date, name) VALUES ((SELECT agency_id FROM agency WHERE name = 'Marlim Ecological Consulting Ltd.'), NOW(), 'Not Applicable');
+  INSERT INTO INVESTMENT_ACTION_CATEGORY (agency_id, record_effective_date, name) VALUES ((SELECT agency_id FROM agency WHERE name = 'Masse Environmental Consultants Ltd.'), NOW(), 'Not Applicable');
+  INSERT INTO INVESTMENT_ACTION_CATEGORY (agency_id, record_effective_date, name) VALUES ((SELECT agency_id FROM agency WHERE name = 'Matrix Solutions Inc.'), NOW(), 'Not Applicable');
+  INSERT INTO INVESTMENT_ACTION_CATEGORY (agency_id, record_effective_date, name) VALUES ((SELECT agency_id FROM agency WHERE name = 'Max Planck Institute, Ploen, Germany'), NOW(), 'Not Applicable');
+  INSERT INTO INVESTMENT_ACTION_CATEGORY (agency_id, record_effective_date, name) VALUES ((SELECT agency_id FROM agency WHERE name = 'McCleary Aquatic Systems Consulting'), NOW(), 'Not Applicable');
+  INSERT INTO INVESTMENT_ACTION_CATEGORY (agency_id, record_effective_date, name) VALUES ((SELECT agency_id FROM agency WHERE name = 'McElhanney Ltd.'), NOW(), 'Not Applicable');
+  INSERT INTO INVESTMENT_ACTION_CATEGORY (agency_id, record_effective_date, name) VALUES ((SELECT agency_id FROM agency WHERE name = 'McGill University, Redpath Museum'), NOW(), 'Not Applicable');
+  INSERT INTO INVESTMENT_ACTION_CATEGORY (agency_id, record_effective_date, name) VALUES ((SELECT agency_id FROM agency WHERE name = 'McTavish Resource & Management Consultants Ltd.'), NOW(), 'Not Applicable');
+  INSERT INTO INVESTMENT_ACTION_CATEGORY (agency_id, record_effective_date, name) VALUES ((SELECT agency_id FROM agency WHERE name = 'Metro Vancouver (GVRD)'), NOW(), 'Not Applicable');
+  INSERT INTO INVESTMENT_ACTION_CATEGORY (agency_id, record_effective_date, name) VALUES ((SELECT agency_id FROM agency WHERE name = 'Metro Vancouver Regional Parks'), NOW(), 'Not Applicable');
+  INSERT INTO INVESTMENT_ACTION_CATEGORY (agency_id, record_effective_date, name) VALUES ((SELECT agency_id FROM agency WHERE name = 'Michigan State University'), NOW(), 'Not Applicable');
+  INSERT INTO INVESTMENT_ACTION_CATEGORY (agency_id, record_effective_date, name) VALUES ((SELECT agency_id FROM agency WHERE name = 'Mid Vancouver Island Habitat Enhancement Society'), NOW(), 'Not Applicable');
+  INSERT INTO INVESTMENT_ACTION_CATEGORY (agency_id, record_effective_date, name) VALUES ((SELECT agency_id FROM agency WHERE name = 'Ministry of Environment & Climate Change Strategy'), NOW(), 'Not Applicable');
+  INSERT INTO INVESTMENT_ACTION_CATEGORY (agency_id, record_effective_date, name) VALUES ((SELECT agency_id FROM agency WHERE name = 'Ministry of Forests'), NOW(), 'Not Applicable');
+  INSERT INTO INVESTMENT_ACTION_CATEGORY (agency_id, record_effective_date, name) VALUES ((SELECT agency_id FROM agency WHERE name = 'Ministry of Forests, Lands, Natural Resource Operations & Rural Development'), NOW(), 'Not   Applicable');
```

Comment:
nit: there's an extra space in 'Not  Applicable'

Reply:
fixed!